protocols/directfund/directfund_test.go

```diff
@@ -119,16 +119,35 @@ func TestUpdate(t *testing.T) {
 
 	// Finally, add some Holdings information to the event
 	// Updating the objective with this event should overwrite the holdings that are stored
-	e.Holdings = types.Funds{}
-	e.Holdings[common.Address{}] = big.NewInt(3)
-	updatedObjective, err = s.Update(e)
+	newFunding := types.Funds{}
+	newFunding[common.Address{}] = big.NewInt(3)
```

Comment:
just fyi (I find this more pleasing, but reasonable people might not):

```suggestion
	newFunding := types.Funds{
		common.Address{}: big.NewInt(3),
	}
```